PODMAN.md

```diff
@@ -0,0 +1,10 @@
+# Use kubevirtci with podman instead of docker
+
+Install podman 3.1+, then run it in docker compatible mode:
```

Comment:
Maybe add that the docker-cli still needs to be there or at least a symlink to podman?
Or is that also no longer the case?

Reply:
And how is the compatibility?
Will it also work to run podman without sudo and point to `unix:///home/jdoe/docker.sock` without breaking anything?
If yes that's what I'd suggest more than the sudo podman.

Reply:
The `docker-cli` should not be needed anymore. Also addressed the `sudo` requirement and documented another required socket location as well as adding the info that `sudo` is not only needed if better storage performance is a must.